src/Neo/Wallets/NEP6/NEP6Wallet.cs

```diff
@@ -223,6 +224,10 @@ public override IEnumerable<WalletAccount> GetAccounts()
 
         public override WalletAccount Import(X509Certificate2 cert)
         {
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
```

Comment:
Can it be implemented with 3rd-party libraries for OSX?

Reply:
have tried, cant extract the private key. nothing in the ecdsa or cert is implementated.

Reply:
OpenSSL

Reply:
not necessary, no one use this. exception is fine already.

Reply:
If `openssl` is installed on the machine and configured right. `dotnet` will use it automatically.

Reply:
but if it works with openssl installed, with this condition it wont never work